### DISCUSSION TITLE
Setting drop down settings with dconf...

### DISCUSSION THREAD

**Discussion by coc0nut** [ddterm/gnome-shell-extension-ddterm#124] [General]
dconf write /com/github/amezin/ddterm/tab-position "['top']"
dconf write /com/github/amezin/ddterm/window-position "['bottom']"

trying to set the dropdown settings with these commands, but they wont change...
only setting i can set is the default one it seems? all options sets the default setting...

**Comment by amezin** [comment]
I'm not that familiar with dconf... Aren't you writing arrays of strings instead of just strings?

    $ dconf read /com/github/amezin/ddterm/tab-position
    'bottom'

i. e. `'bottom'`, not `['bottom']`

**Reply by coc0nut** [reply]
nope, I tried quite some different things... dconf write /com/github/amezin/ddterm/window-position '0' - tried 0, 1, 2, 3
tried bottom, 'bottom', ['bottom'], "['bottom']" - the last one here seems to be the right syntax... but.. this is actually only because I didnt find the schema in gsettings...

**Reply by amezin** [reply]
`dconf write /com/github/amezin/ddterm/window-position "'bottom'"` moves ddterm window to the bottom position for me.
As far as I understand, dconf read/write should work without schema. Schema is applied in GSettings, i. e. on top of dconf, by the application/GLib code.

**Reply by coc0nut** [reply]
that made it! thx! :) tried all things except that I think :p
